sfa_dash/form_utils/converters.py

```diff
@@ -400,6 +400,131 @@ def parse_api_filters(cls, report_parameters):
             'quality_flags': quality_flags
         }
 
+    @classmethod
+    def extract_tods(cls, value):
+        """Parses time of day values from a csv string"""
+        return value.split(',')
+
+    @classmethod
+    def extract_datetimes(cls, value):
+        return value.split(',')
+
+    @classmethod
+    def extract_many_costs(cls, value):
+        costs = value.split(',')
+        costs = [float(cost) for cost in costs]
+        return costs
```

Comment:
for all of these `extract_`, when empty '' is passed so at least `float` raises an error

Reply:
> Is white space allowed in the csv fields?
> 
> Can the bands be reordered once created? Maybe something for future work.

Whitespace should be allowed, but I'll test it out.
Bands can't be reordered right now, but I agree valuable feature for the future. I'm not sure how simple it would be to implement.